snuba/manual_jobs/runner.py

```diff
@@ -84,10 +98,12 @@ def list_job_specs_with_status(
     manifest_filename: str = MANIFEST_FILENAME,
 ) -> Mapping[str, Mapping[str, Union[JobSpec, JobStatus]]]:
     specs = list_job_specs(manifest_filename)
-    job_ids = specs.keys()
+    job_ids = list(specs.keys())
     statuses = _get_job_status_multi(
         [_build_job_status_key(job_id) for job_id in job_ids]
     )
+    for i in range(len(job_ids)):
```

Comment:
I know this for loop defeats the purpose of mget used in `_get_job_status_multi`, so I'm also not opposed to write `_get_is_job_async_multi` and `_get_job_types_multi`

Reply:
What would the implementations of `_get_is_job_async_multi` and `_get_job_types_multi` look like?

Reply:
We got rid of `[get_is_job_async](https://github.com/getsentry/snuba/pull/6427#discussion_r1809149842)`, so I won't be implementing `_get_is_job_async_multi`. The implementation of `_get_job_types_multi` will just mirror `_get_job_types_multi` and `_get_job_status_multi`

Reply:
^ will revisit this in subsequent PR